code/modules/admin/verbs/deadsay.dm

```diff
@@ -25,14 +25,8 @@
 		rank_name = pick(strings("admin_nicknames.json", "ranks", "config"))
 		admin_name = pick(strings("admin_nicknames.json", "names", "config"))
 	var/name_and_rank = "[span_tooltip(rank_name, "STAFF")] ([admin_name])"
-	var/rendered = "<span class='game deadsay'>[span_prefix("DEAD:")] [name_and_rank] says, <span class='message'>\"[emoji_parse(msg)]\"</span></span>"
-
-	for (var/mob/M in GLOB.player_list)
-		var/admin_holder = M.client?.holder
-		if(isnewplayer(M) && !admin_holder) // We want to make sure admins can see this when in the lobby too!
-			continue
-		if (M.stat == DEAD || (admin_holder && (M.client?.prefs.chat_toggles & CHAT_DEAD))) //admins can toggle deadchat on and off. This is a proc in admin.dm and is only given to Administrators and above
-			to_chat(M, rendered, confidential = TRUE)
+
+	deadchat_broadcast("[span_prefix("DEAD:")] [name_and_rank] says, <span class='message'>\"[emoji_parse(msg)]\"</span>")
```

Comment:
Just making sure, this doesn't add a (F) button to follow the admins to that message, yeah? It's kind of important it doesn't, even if they're observing, in my opinion, to avoid players just swarming staff whenever they speak in dsay.

Reply:
https://github.com/tgstation/tgstation/blob/491107d56399a36dd25ae968fac94972070b4383/code/__HELPERS/mobs.dm#L529

follow_target and turf_target are by default null so the (F) button shouldn't be added.